File: sdks/thunkmetrc-go/wrapper/models/packages_in_transit_product_label.go
package models

type PackagesInTransitProductLabel struct {
    IsActive bool `json:"IsActive,omitempty"`
    IsChildFromParentWithLabel bool `json:"IsChildFromParentWithLabel,omitempty"`
    LabelSource *string `json:"LabelSource,omitempty"`
    LastLabelGenerationDate *string `json:"LastLabelGenerationDate,omitempty"`
    OriginalSourcePackageId *int64 `json:"OriginalSourcePackageId,omitempty"`
    OriginalSourcePackageLabel *string `json:"OriginalSourcePackageLabel,omitempty"`
    QrCodeRanges *string `json:"QrCodeRanges,omitempty"`
    QrCount int64 `json:"QrCount,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesInTransitProductLabel.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesInTransitProductLabel(
    @JsonProperty("IsActive")
    val isActive: Boolean? = null,
    @JsonProperty("IsChildFromParentWithLabel")
    val isChildFromParentWithLabel: Boolean? = null,
    @JsonProperty("LabelSource")
    val labelSource: String? = null,
    @JsonProperty("LastLabelGenerationDate")
    val lastLabelGenerationDate: String? = null,
    @JsonProperty("OriginalSourcePackageId")
    val originalSourcePackageId: Int? = null,
    @JsonProperty("OriginalSourcePackageLabel")
    val originalSourcePackageLabel: String? = null,
    @JsonProperty("QrCodeRanges")
    val qrCodeRanges: String? = null,
    @JsonProperty("QrCount")
    val qrCount: Int? = null
)
